samples/go/main.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 // #cgo CFLAGS: -I../../include
-// #cgo LDFLAGS: -L../../build -L../../build/trezor-crypto -lTrustWalletCore -lprotobuf -lTrezorCrypto -lc++ -lm
+// #cgo LDFLAGS: -L../../build -L../../build/trezor-crypto -lTrustWalletCore -lprotobuf -lTrezorCrypto -lstdc++ -lm
```

Comment:
does this change work for both Ubuntu and Mac?

Reply:
Yes.  Clang is used in both places.